pkg/timeutil/timeutil.go

```diff
@@ -20,9 +24,32 @@ func Date(datetime time.Time) time.Time {
 func LastWeekday(weekday time.Weekday) time.Time {
 	date := Date(time.Now())
 
-	for date.Weekday() != weekday {
+	for date.Weekday() !=  weekday{
 		date = date.AddDate(0, 0, -1)
 	}
 
 	return date
 }
+
+// StringToWeekday transforms a string to time.Weekday
+// it returns Monday if wrong day specified in config.toml
+func StringToWeekday(weekday string) (time.Weekday, error){
```

Comment:
I'd leave this one as is, but I said to Callum to post his solution because it's an interesting alternative. The switch statement is faster because it's doing less things. This is shorter and doesn't use the "magic" strings like in the switch.

There may be similar things in the future though that benefit from the approach that Callum has taken here, e.g. if there were 20 different days in a week then that would be a LONG switch statement...